skimage/feature/_texture.pyx

```diff
@@ -13,7 +13,7 @@ cdef extern from "numpy/npy_math.h":
     double NAN "NPY_NAN"
 
 
-def _glcm_loop(cnp.uint8_t[:, ::1] image, double[:] distances,
+def _glcm_loop(cnp.uint16_t[:, ::1] image, double[:] distances,
```

Comment:
Perform the corresponding change in the docstring. At the moment, it is `Input image, which is converted to the _uint8_ data type.`.